prime-angular-frontend/src/app/modules/paper-enrolment/pages/care-setting-page/overview-care-setting/overview-care-setting.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { Router, ActivatedRoute } from '@angular/router';
+
+import { RouteUtils } from '@lib/utils/route-utils.class';
+
+import { PaperEnrolmentRoutes } from '@paper-enrolment/paper-enrolment.routes';
+@Component({
+  selector: 'app-overview-care-setting',
+  templateUrl: './overview-care-setting.component.html',
+  styleUrls: ['./overview-care-setting.component.scss']
+})
+export class OverviewCareSettingComponent implements OnInit {
```

Comment:
First thing I notice is that the overview component is in 4 files not a single file with a spec, and the name of the file has overview first instead of last - `care-setting-overview.component.ts`.  With the CLI you can specify inline style and template.